DFC.Digital/DFC.Digital.Service.CourseSearchProvider.UnitTests/Helpers/MemberDataHelper.cs

```diff
@@ -0,0 +1,277 @@
+﻿using DFC.Digital.Data.Model;
+using DFC.Digital.Service.CourseSearchProvider.CourseSearchServiceApi;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DFC.Digital.Service.CourseSearchProvider.UnitTests
+{
+    public class MemberDataHelper
+    {
+        private const string NormalWorkingHoursPattern = "AP1";
```

Comment:
Is there a reason for these consts here and not using the other constants class?